rpm/fedora-30/Dockerfile

```diff
@@ -0,0 +1,23 @@
+ARG GO_IMAGE
+ARG ENGINE_IMAGE
+ARG BUILD_IMAGE=fedora:30
+FROM ${GO_IMAGE} as golang
+FROM ${ENGINE_IMAGE} as engine
+
+FROM ${BUILD_IMAGE}
+ENV DISTRO fedora
+ENV SUITE 30
```

Comment:
Could put this in an `ARG`, then

```
ARG BUILD_IMAGE=fedora:${SUITE}
```

or

```
FROM ${BUILD_IMAGE:-fedora:${SUITE}}
```

Reply:
It could but I'd rather not have a special process specifically for `fedora` that's different from the other RPM distributions we support (both CE and EE)

Reply:
you'd still set the `BUILD_IMAGE` arg to specify the image, only you get rid of the double `30`